pkg/backupapi/file.go

```diff
@@ -88,3 +97,95 @@ func (c *Client) UploadFile(fn string, r io.Reader, pw io.Writer) error {
 	_, err = io.Copy(ioutil.Discard, resp.Body)
 	return err
 }
+
+func (c *Client) uploadMultipart(recoveryPointID string, r io.Reader, pw io.Writer) error {
+	ctx := context.Background()
+	m, err := c.InitMultipart(ctx, recoveryPointID)
+	if err != nil {
+		return err
+	}
+
+	partNum := 0
```

Comment:
The part number begin 1, not 0. I'll update the api docs.

Reply:
See the for loop, `partNum++` at the beginning, also see the test, which verify that `part_number` is greater than 0.